starfield.js

```diff
@@ -0,0 +1,101 @@
+/*
+	Starfield lets you take a div and turn it into a starfield.
+
+*/
+
+//	Define the starfield class.
+function Starfield() {
+	this.fps = 30;
+	this.canvas = null;
+	this.width = 0;
+	this.height = 0;
+	this.minVelocity = 15;
+	this.maxVelocity = 30;
+	this.stars = 100;
```

Comment:
```suggestion
	this.stars = 200;
```

I could imagine a few more stars.

Reply:
this does not look good on small screens / mobile.

so, unless we adapt that somehow, i suggest to stay with the current default - that is "starry" on mobile and "decent" on desktop.